python/src/etos_api/routers/etos/router.py

```diff
@@ -142,6 +144,32 @@ async def _start(etos: StartEtosRequest, span: "Span") -> dict:
     }
 
 
+async def _abort(suite_id: str) -> dict:
+    ns = namespace()
+
+    batch_api = client.BatchV1Api()
+    jobs = batch_api.list_namespaced_job(namespace=ns)
+
+    delete_options = client.V1DeleteOptions(
+        propagation_policy="Background"  # asynchronous cascading deletion
+    )
+
+    for job in jobs.items:
+        if (
+            job.metadata.labels.get("app") == "suite-runner"
+            and job.metadata.labels.get("id") == suite_id
+        ):
+            batch_api.delete_namespaced_job(
+                name=job.metadata.name, namespace=ns, body=delete_options
+            )
+            LOGGER.info("Deleted suite-runner job: %s", suite_id)
```

Comment:
It would be more convenient if this log had a reference to the complete jobname not just the suite ID. (Which is only a part of the jobname)

Reply:
Done